tests/uefistored/test_auth_var.py

```diff
@@ -3,6 +3,8 @@
 from lib.commands import SSHCommandFailed
 from lib.efi import (
     Certificate,
+    EFIAuth,
```

Comment:
Maybe `EFI_AUTH` to use a unified syntax?

Reply:
Unfortunately `EFIAuth` is a class and so should probably stay CamelCase, as opposed to the `UPPER_SNAKECASE` variables, which are just "constant" primitives.

I would like to eventually replace the GUIDs with an object that has `as_str()` and `as_bytes()` methods, which will clean up the verbose exports, but that's probably for another PR...

Reply:
Oh! I misread the code. :sweat_smile: